x/oracle/keeper/native_token.go

```diff
@@ -21,7 +21,26 @@ import (
 // msg(refund or slash on beaconChain): update staker's price, operator's price
 
 const (
-	NSTETHASSETID = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65"
+	// TODO: we currently support NSTETH only which has capped effective balance for one validator
+	// TODO: this is a bad practice, and for Lz, they have different version of endpoint with different chainID
+	// Do the validation before invoke oracle ralted functions instead of check these hard code ids here.
+	NSTETHAssetIDMainnet = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65"
```

Comment:
mark 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65 as localnet. Mainnet should be 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x7595

Reply:
updated